src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -168,9 +168,8 @@ function MoneyRequestPreview(props) {
     const isDeleted = lodashGet(props.action, 'pendingAction', null) === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE;
 
     // Show the merchant for IOUs and expenses only if they are custom or not related to scanning smartscan
-    const shouldShowMerchant =
-        !_.isEmpty(requestMerchant) && !props.isBillSplit && requestMerchant !== CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT && requestMerchant !== CONST.TRANSACTION.DEFAULT_MERCHANT;
-    const shouldShowDescription = !_.isEmpty(description) && !shouldShowMerchant;
+    const shouldShowMerchant = !_.isEmpty(requestMerchant) && requestMerchant !== CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT && requestMerchant !== CONST.TRANSACTION.DEFAULT_MERCHANT;
+    const merchantOrDescription = shouldShowMerchant ? requestMerchant : description || '';
```

Comment:
Our design is we either show the merchant if it's not the default one or empty, otherwise we show the description.

Reply:
cc @trjExpensify @shawnborton this is going to fix the merchant having a small font size as well

<img width="430" alt="Screenshot 2023-11-14 at 22 17 09" src="https://github.com/Expensify/App/assets/9680864/e036b34d-5402-45f8-b934-a643c4e854f3">

Reply:
That looks good to me, but I'll defer to Shawn.

Reply:
That looks good to me. Can you also take a look at [this one](https://expensify.slack.com/archives/C049HHMV9SM/p1700000146978699) while you are here? Thanks!